Cubical/Algebra/CommAlgebra/LocalisationAlgebra.agda

```diff
@@ -0,0 +1,254 @@
+{- Localisation of an algebra at an arbitrary multiplicatively closed subset.
+
+There are multiple ways one might want to go about formalizing this, from simply
+extending the ring construction mechanically to using the fact the category of
+algebras is a coslice category and exactly re-using the localization of rings.
+Here, we chose the former, because after trying out the latter for some time, it
+turned out that its computational behavior wasn't as nice as expected, and it
+made Agda struggle a lot.
+-}
+
+{-# OPTIONS --safe #-}
+
+open import Cubical.Core.Everything
```

Comment:
Probably not needed.

Reply:
I've removed all of those unnecessary imports.